yamls/ndm-operator.yaml

```diff
@@ -461,7 +461,7 @@ spec:
       #hostPID: true
       containers:
       - name: node-disk-manager
-        image: openebs/node-disk-manager:1.6.1
+        image: openebs/node-disk-manager:1.7.0
         args:
           - -v=4
         # The feature-gate is used to enable the new UUID algorithm.
```

Comment:
The flag needs to be renamed to ChangeDetection on [line](https://github.com/openebs/charts/pull/282/files?file-filters%5B%5D=.yaml&hide-deleted-files=true#diff-3669f197ed8f059697b8c8e73444abb5dbff592666cb33180040cc9a439c835aR471)